src/map/script.c

```diff
@@ -23049,10 +23051,11 @@ static BUILDIN(pushpc)
 	}
 	else if(cells<0)
 	{// pushing backwards
-		dir = (dir+4)%8;  // turn around
-		cells     = -cells;
+		dir   = unit_get_opposite_dir(dir);
```

Comment:
Remove superfluous space in front of equality sign.

Reply:
It's so dir = and cells = equation signs are on same level. Won't change.